dan-bock-stpaul.dan

```diff
@@ -0,0 +1,8 @@
+# St Paul, MN
+
+### Restaurants to Visit
+
```

Comment:
You should add a restaurant here, that would make this better, and not garbage. :smile: